tests/core/test_structured_data_signing.py

```diff
@@ -764,11 +764,14 @@ def test_hashed_structured_data_with_nested_structs():
 @pytest.mark.parametrize(
     "type, valid",
     (
-        ("unint bytes32", False),
         ("hello\\[]", False),
         ("byte[]uint", False),
         ("byte[7[]uint][]", False),
         ("Person[0]", False),
+        (" messageType", True),
```

Comment:
I see the valid message json file has multiple spaces so not sure if it would add value to include a couple other scenarios here with multiple spaces. Ones I would pick:
```suggestion
        (" messageType ", True),
        ("  messageType", True),
```

Reply:
Good recommendations. Will add.